internal/pkg/test/privilege_darwin.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package test
```

Comment:
To be honest, I think all of these can just be no-op functions for Darwin. I can't (yet) think of a scenario where we need do testing in this way on Darwin